src/Components/Web.JS/src/Platform/Mono/MonoDebugger.ts

```diff
@@ -3,23 +3,27 @@
 
 import { WebAssemblyResourceLoader } from '../WebAssemblyResourceLoader';
 
+const navigatorUA = navigator as MonoNavigatorUserAgent;
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const currentBrowserIsChrome = (window as any).chrome
-  && navigator.userAgent.indexOf('Edge') < 0; // Edge pretends to be Chrome
+const currentBrowserIsChromeOrEdge = navigatorUA.userAgentData && navigatorUA.userAgentData.brands ?
+     (navigatorUA.userAgentData.brands.some(b => b.brand === 'Google Chrome') ||
+      navigatorUA.userAgentData.brands.some(b => b.brand === 'Microsoft Edge'))
+      : (window as any).chrome && navigator.userAgent.indexOf('Edge') < 0; // Edge pretends to be Chrome
```

Comment:
Just a suggestion to make it more readable:

```suggestion
const brands = navigatorUA.userAgentData && navigatorUA.userAgentData.brands;
const currentBrowserIsChromeOrEdge = brands
  ? brands .some(b => b.brand === 'Google Chrome' || b.brand === 'Microsoft Edge')
  : (window as any).chrome;
```

I'm not sure we should still be concerned with `navigator.userAgent.indexOf('Edge')` because modern Edge doesn't include the string `Edge` in its useragent (it has `Edg` instead to avoid these checks). And very old versions of Edge aren't really a concern any more for development time.

Reply:
Thanks Steve for the review, I have addressed your feedback accordingly (I hope).